sdk/nestjs/src/DevCycleModule/DevCycleModule.ts

```diff
@@ -0,0 +1,41 @@
+import { Module, DynamicModule, Global } from '@nestjs/common'
+import { APP_INTERCEPTOR } from '@nestjs/core'
+import { initializeDevCycle, DevCycleClient, DevCycleOptions } from '@devcycle/nodejs-server-sdk'
+import { RequestInterceptor } from './RequestInterceptor'
+
+@Global()
```

Comment:
is it ok for a lib to force itself to be global like this? or is it more expected that the end user can decide? I believe I've seen other libs take a "global" setting in the `forRoot` method that decides it